src/python/pants/core/util_rules/subprocess_environment.py

```diff
@@ -22,11 +22,13 @@
     "ftp_proxy",
     "all_proxy",
     "no_proxy",
+    "requests_ca_bundle",
```

Comment:
IIUC there is no need for lowercase here. The preceding block does need both because the standard was oddly set that way, but that's an exception. See and maybe add a note / link for this blocks odd double casing: https://www.w3.org/Library/User/History.html#z13

Reply:
I thought so. I'll remove that then!